binding/python/test_orca.py

```diff
@@ -0,0 +1,135 @@
+#
+#    Copyright 2024 Picovoice Inc.
+#
+#    You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+#    file accompanying this source.
+#
+#    Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+#    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#    specific language governing permissions and limitations under the License.
+#
+
+import argparse
```

Comment:
so we dont have any test to check the generated pcm, right? Can we use Leopard to calculate WER/CER on it?

Reply:
Added leopard to the python sdk tests